codersdk/database.go

```diff
@@ -0,0 +1,7 @@
+package codersdk
+
+import "errors"
+
+const DatabaseNotReachable = "database not reachable"
+
+var ErrDatabaseNotReachable = errors.New(DatabaseNotReachable)
```

Comment:
Self-review: not using anywhere yet, since we can't pass _actual_ sentinel errors around in this case, but for other use-cases it will be useful. Happy to remove, though.